### PR TITLE
Requestrr Docs

### PR DESCRIPTION
**Pull request by HarleyL7**
Added docs for requestrr

### CODE REVIEW

File: docs/applications/requestrr.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  2. Still in the Overseerr settings page, copy the API Key
+  3. Paste the API Key into Requestrr API Key Field
+  4. Enter the hostname or IP, If Requestrr and Radarr are on the same machine then use `127.0.0.1`
+  5. Press Test Settings at the bottom of the page
+  6. Save your changes
+
+More information in regards to permissions and linking a Discord user to an Overseerr user can be found [Here](https://github.com/darkalfx/requestrr/wiki/Configuring-Overseerr#configuring-permissions)
+
+ </TabItem>
+
```
Comment (brettpetch) [inline docs/applications/requestrr.mdx:127]: you need to close tabs. see this for an example: 
https://pipelines.actions.githubusercontent.com/etBdarlm6eMDwSa1NkNZo2fsgjLVPIg6bOIIEVHCrEhB8zbABn/_apis/pipelines/1/runs/153/signedlogcontent/4?urlExpires=2021-03-22T04%3A33%3A59.4675604Z&urlSigningMethod=HMACV1&urlSignature=5qZUA7K5FB0aFQlBVY7aw6IzQhf3cr%2F8ZeTGcCdTCqw%3D

File: docs/applications/requestrr.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+<Tabs
+  defaultValue="Radarr"
+  values={[
+    {label: 'Radarr', value: 'radarr'},
+    {label: 'Ombi', value: 'ombi'},
+    {label: 'Sonarr', value: 'sonarr'},
+    {label: 'Overseerr', value: 'overseerr'},
+  ]}>
+<TabItem value="radarr">
```
Comment (brettpetch) [inline docs/applications/requestrr.mdx:69]: should be indented, as is a tag under tabs. others too.

**Review by brettpetch — CHANGES_REQUESTED** [review]

**Comment by brettpetch** [comment]
@HarleyL7 you may need to modify your branch a bit... you forgot to add docs for the application to the sidebar.

/b

**Comment by flying-sausages** [comment]
++ the frontpage list of applications

**Pull request closed without merging**